x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/alerts/basic_license_essentials_tier/set_alert_tags.ts

```diff
@@ -40,8 +40,8 @@ export default ({ getService }: FtrProviderContext) => {
   const dataPathBuilder = new EsArchivePathBuilder(isServerless);
   const path = dataPathBuilder.getPath('auditbeat/hosts');
 
-  // Intentionally setting as @skipInQA, keeping tests running in MKI that should block release
-  describe('@ess @serverless @skipInQA set_alert_tags', () => {
+  // Intentionally setting as @skipInServerless, keeping tests running in MKI that should block release
+  describe('@ess @serverless @skipInServerless set_alert_tags', () => {
```

Comment:
This one's a bit tricky - we technically need to add `serverlessQA` to any test suites that **do not** have `@skipInQA` and remove `@skipInQA` from test suites that have this comment about intentionally setting @skipInQA, because we do want those run in serverless, just not in MKI.

Reply:
> because we do want those run in serverless, just not in MKI.

Per the agreement we did on the Cypress PR: https://github.com/elastic/kibana/pull/179737 this is not possible without having an extra tag. Currently for both API and cypress:
- A serverless tests is executed in CI, periodic pipeline (MKI) and release process (MKI)
- A serverless tests is executed in CI and periodic pipeline (MKI)
- A serverless test is not executed